public_html/quickstart.php

```diff
@@ -426,6 +426,21 @@
 					</div>
 				</div>
 			</div>
+
+			<div class="container-con-block darkmode-block">
+				<div class="anchorpoint" id="dumping_linux"></div>
+				<div class='container-con-wrapper'>
+					<div class="container-tx1-block darkmode-txt">
+						<h2>A command-line option for Linux users</h2>
+					</div>
+					<div class="container-tx2-block darkmode-txt">
+						<p>
+							If you're comfortable with the Linux command-line and you have a compatible BluRay drive, you can try ripping PS3 discs using a Python program called <a href="https://notabug.org/necklace/libray">LibRay</a>.

```

Comment:
little formatting changes
```suggestion
							If you're comfortable with the Linux command-line and you have a compatible BluRay drive, you can try ripping PlayStation 3 discs using a Python program called <a href="https://notabug.org/necklace/libray" target="_blank" rel="noopener noreferrer">LibRay</a>.
```